### PR TITLE
update doc to use new customization yaml format

### PR DESCRIPTION
**Pull request by pruan-rht**
docs/user/aws: update customization for controlPlane/compute

The machine field in install-config.yaml has been replaced with the controlPlane
and compute fields. The docs/user/aws/customization.md file needs to be updated
to reflect the change in the structure of install-config.yaml.

### CODE REVIEW

File: docs/user/aws/customization.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ An example `install-config.yaml` is shown below. This configuration has been mod
 ```yaml
 apiVersion: v1beta1
```
Comment (abhinavdahiya) [inline docs/user/aws/customization.md]: `v1beta3`

Comment (pruan-rht) [reply docs/user/aws/customization.md]: @abhinavdahiya, good catch...updated.

**Pull request merged**